src/lib/datepicker/datepicker.ts

```diff
@@ -186,6 +188,8 @@ export class MatDatepicker<D> extends CdkDatepicker<D> implements OnDestroy, Can
   set opened(value: boolean) { value ? this.open() : this.close(); }
   private _opened = false;
 
+  @ViewChild(MatCalendar) calTemplate: MatCalendar<D>;
```

Comment:
`calTemplate` -> `calendar` (it's not actually a template)

Also looks like it should be prefixed with an underscore and have a JsDoc description